FirebaseFirestoreSwift.podspec

```diff
@@ -0,0 +1,37 @@
+#
+# Be sure to run `pod lib lint FirebaseFirestoreSwift.podspec' to ensure this is a
+# valid spec before submitting.
+#
+
+Pod::Spec.new do |s|
+  s.name                    = 'FirebaseFirestoreSwift'
+  s.version                 = '0.10.1'
```

Comment:
Might want to explicitly version this separately since it presumably doesn't need to stay in lock step.

Reply:
Done.